numpy/lib/function_base.py

```diff
@@ -3948,7 +3948,8 @@ def _quantile_ureduce_func(a, q, axis=None, out=None, overwrite_input=False,
             r_above = r_above.squeeze(0)
             r_below = r_below.squeeze(0)
 
-        r_above[r_above < 0.5] = r_below[r_above < 0.5]
+        # TODO: use the out argument to `np.where` once we add it (gh-XXXX)
```

Comment:
The comment is not really relevant here anymore. `np.where(condition, x, y, out=x)` seems equivalent to `np.copyto(x, y, where=condition)`. So that is the nicer solution here in any case.

Reply:
Oh I see, I thought that you still wanted to change for `np.where`. Not really sure about the windows failure.